app/templates/guide.twig

```diff
@@ -18,7 +19,22 @@
                 </div>
             {% endif %}
 
-            {{ guide.renderedContent|raw }}
+            {% if guide.sections|length %}
+                {% set links = [] %}
+                {% for section in guide.sections %}
+                    {% if section.children|length %}
+                        {% for subsection in section.children %}
+                            {% if subsection.id in links %}
+                                {% set anchorLinks = anchorLinks|merge([{ parent: section.id, sub: subsection.id,  }]) %}
```

Comment:
I'd be more descriptive with the array keys and call them `parentId` (or `sectionId`) and `subsectionId`.

Reply:
Updated